hack/vgopath-setup.sh

```diff
@@ -1,14 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
I believe this script is always sourced. I.e., it doesn't need a shebang line.

Reply:
Yes, it is always sourced. I added the shebang to fix a linter warning. The license line starts with `#` too, so the linter complains about an unknown shell 😉